feat(rqt): Add update rate and async info to details window

ShahazadAbdulla opened the pull request:
Fixes #2166

This PR adds the `update_rate` and `is_async` parameters to the RQT controller manager details popup window.

**Implementation Details:**
- Modified `popup_info.ui` to add new QLabel widgets for displaying the information.
- Updated the `_on_ctrl_info` function in `controller_manager.py` to fetch the parameters using `ros2param.api.call_get_parameters`.
- The logic first attempts to retrieve per-controller parameters (`<controller_name>.update_rate`, etc.).
- If a per-controller value is not found, it gracefully falls back to the global `update_rate` or a sensible default (`False` for `is_async`).

**Testing:**
- The feature was developed and tested in a ROS 2 Rolling Docker environment.
- When running the `rrbot` demo from `ros2_control_demos`, the popup correctly displays the global and default values as shown below.

**Result:**

<img width="1920" height="1061" alt="screenshot-20250902_135959" src="https://github.com/user-attachments/assets/fc9d7a42-8dd3-40d6-948f-5bc6fa47763d" />

<img width="639" height="1057" alt="screenshot-20250902_140045" src="https://github.com/user-attachments/assets/722024cc-f990-482b-8f45-cdb31b833be9" />

christophfroehlich reviewed (changes requested):
Don't worry about this failing test, this is not related. But you haven't addressed my concern regarding update of the hardware components, please do so.

ShahazadAbdulla commented:
Hi @christophfroehlich 

Thank you for the clarification. I have now pushed the final fix.

I've updated both the controller and hardware component info dialogs to reset their labels to "N/A" on opening. This prevents stale data from being shown and fully addresses your review feedback. I have also run pre-commit to format the changes.

Thank you again for your patience and guidance.
